docs/docs/gestures/_shared/base-gesture-event-data.md

```diff
@@ -7,3 +7,11 @@ Current [state](/docs/fundamentals/states-events) of the handler. Expressed as o
 ### `numberOfPointers`
 
 Represents the number of pointers (fingers) currently placed on the screen.
+
+### `pointerType`
+Indicates the type of pointer device being used. This value is represented by one of the constants defined in the `PointerType` enum, which includes the following fields:
```

Comment:
```suggestion
Indicates the type of pointer device in use. This value is represented by the `PointerType` enum, which includes the following fields:
```

You should strongly prefer writing technical documentation in an **active voice**. More info: https://developers.google.com/tech-writing/one/active-voice

The second sentence is a bit harder to write in active voice so I made it shorter.